sql/schema.go

```diff
@@ -103,3 +103,21 @@ func IsKeyless(s Schema) bool {
 
 	return true
 }
+
+type PrimaryKeySchema struct {
+	Schema
+	pkOrdinals []int
+}
+
+func NewPrimaryKeySchema(s Schema, pkOrds []int) PrimaryKeySchema {
```

Comment:
Suggest making pkOrds a `...` varargs param, and change the behavior to automatically look for primary keys in the schema given if no ords are provided. Then you can avoid passing []int{} in all the test files.

Alternately, create a separate method, such as `PKSchemaFromSchema(schema)` that behaves the way I'm describing. And document how the method behaves in a comment.

Reply:
f

Reply:
not making that mistake again